modules/get-started/pages/licensing/add-license-redpanda/linux.adoc

```diff
@@ -0,0 +1,71 @@
+= Add an Enterprise Edition License to Redpanda in Linux
+:description: Learn how to add or update a Redpanda Enterprise Edition license in a Linux environment.
+
+To enable xref:get-started:licensing/overview.adoc[enterprise features for Redpanda Self-Managed], it must have an Enterprise Edition license. This guide outlines how to apply or update an Enterprise Edition license for Redpanda Self-Managed in a Linux environment.
+
+== Prerequisites
+
+- You must have xref:get-started:rpk-install.adoc[`rpk` installed] and configured to connect to your Redpanda cluster.
+
+- You must have an Enterprise Edition license. If you don't have a license yet, https://www.redpanda.com/contact[request one^].
+
+== Add a new license
+
+Redpanda supports the following ways to apply a new license:
+
+- <<file, Provide the path to a file containing the license>>.
+- <<inline, Pass the license string directly. Follow the instructions below based on your preferred method>>.
```

Comment:
I'm not sure why you need this extra text? "Follow the instructions below based on your preferred method"